db/migrate/20210601125525_ensure_ems_storage_features.rb

```diff
@@ -0,0 +1,35 @@
+class EnsureEmsStorageFeatures < ActiveRecord::Migration[6.0]
+  class MiqUserRole < ActiveRecord::Base
+    has_and_belongs_to_many :miq_product_features, :join_table => :miq_roles_features, :class_name => "EnsureEmsStorageFeatures::MiqProductFeature"
+  end
+
+  class MiqProductFeature < ActiveRecord::Base
+    has_and_belongs_to_many :miq_user_roles, :join_table => :miq_roles_features, :class_name => "EnsureEmsStorageFeatures::MiqUserRole"
+  end
+
+  def up
```

Comment:
We've been quick-gating these kinds of data migration prs with a `return if MiqProductFeautre.none?` to avoid the empty database going through all of this unnecessarily